sstables/storage.cc

```diff
@@ -556,10 +556,15 @@ sstring s3_storage::make_s3_object_name(const sstable& sst, component_type type)
     if (!sst.generation().is_uuid_based()) {
         throw std::runtime_error("'S3' STORAGE only works with uuid_sstable_identifier enabled");
     }
-    if (sst.is_uploaded()) {
-        return format("/{}/{}", _bucket, sst.filename(type));
-    }
-    return format("/{}/{}/{}", _bucket, sst.generation(), sstable_version_constants::get_component_map(sst.get_version()).at(type));
+
```

Comment:
> The s3_storage backend driver has a method that generates object path
withing the bucket

what does "withing" the bucket mean? do you mean "with" or "within"? i searched this word at [w-m](https://www.merriam-webster.com/dictionary/withing), no luck. i think it should be the latter.  just wanted to check with you.

Reply:
Should be "within"